test/linalg/givens.jl

```diff
@@ -1,5 +1,4 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
-
```

Comment:
no big deal since we'll put it back with an automated script every so often, but the whitespace after the opening license header is considered significant by the current version of `contrib/add_license_to_files.jl`

(best combination of labels)